code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -898,7 +898,7 @@
 			/obj/item/storage/pouch/firstaid = -1,
 			/obj/item/storage/pouch/syringe = -1,
 			/obj/item/storage/pouch/medkit = -1,
-			/obj/item/storage/pouch/med_lolipops = -1,
```

Comment:
This'll put infinite full pouches in the surplus gear vendors, you should probably change the ones in code/_DEFINES/loadout instead.

Reply:
My pr is to fix the broken lolipop pouches. Not balance change it. It was intended to be infinite. So I should not be touching it.

Reply:
I believe those are not intended to be infinite only the ghhme ones should start full